app/swagger/requests/sessions.rb

```diff
@@ -5,34 +5,51 @@ module Requests
     class Sessions
       include Swagger::Blocks
 
-      swagger_path '/v0/sessions/authn_urls' do
```

Comment:
I feel uncomfortable removing this, even though I know vets-website was the only consumer of the endpoint. How do you feel about it?